src/rdkafka_cgrp.c

```diff
@@ -2649,7 +3246,8 @@ rd_kafka_cgrp_incremental_unassign (rd_kafka_cgrp_t *rkcg,
 
         for (i = 0 ; i < partitions->cnt ; i++) {
```

Comment:
let's get it merged! and increb rebased!

Reply:
Fixed.

Reply:
i don't think it helps (much?). also, the code i have is terse and easy to read. i think it's better than jumping into a method that's not a standard building block, which I would need to do if reading this because a. it's non standard and b. potentially dealing with a ref counted field, and i'd want to closely verify behavior around that.

Reply:
The sole purpose of update_toppars() is to do what you are doing here, why reimplement it here now when it is available?

Reply:
i misunderstood, changing - thanks.